mw_jwt_test.go

```diff
@@ -86,39 +87,84 @@ func createJWTSessionWithRSAWithPolicy(policyID string) *user.SessionState {
 	return session
 }
 
-// JWTSessionHMAC
+type JwtCreator func() *user.SessionState
+
+var createHmacJWKSession = func() *user.SessionState { return createJWTSession() }
+var createRsaJWKSession = func() *user.SessionState { return createJWTSessionWithRSA() }
+
+func prepareGenericJWTSession(testName string, method string, claimName string, GlobalSkipKidAsId bool, ApiSkipKidAsId bool) (*APISpec, string) {
+
+	tokenKID := testKey(testName, "token")
+
+	globalConf := config.Global()
```

Comment:
since we change config here - we should not forget to do `defer resetTestConfig()` in all tests where this function is being called (to bring config to initial state)

Reply:
ok